src/main/java/fr/insalyon/creatis/gasw/plugin/executor/dirac/execution/DiracJdlGenerator.java

```diff
@@ -129,6 +141,48 @@ public String generate(String scriptName, Map<String, String> envVariables) {
         }
     }
 
+    public String generate(String scriptName, Map<String, String> envVariables, boolean isMoteurLiteEnabled) {
```

Comment:
do not duplicate the method, re-use the existing one.
if moteurLite is enabled, add the new variables, and use them in the jdl template only in the moteurLite case with a if.